src/modules/10-common/components/EntityReference/EntityReference.tsx

```diff
@@ -287,7 +287,7 @@ export function EntityReference<T>(props: EntityReferenceProps<T>): JSX.Element
             recordRender={recordRender}
             collapsedRecordRender={collapsedRecordRender}
             selectedScope={selectedScope}
-            pagination={props.pagination}
+            pagination={{ ...props.pagination, hidePageNumbers: true }}
```

Comment:
It might be an idea to selectively hide the page numbers only if they don't fit. I find that in my dialogs I can usually fit 4 page numbers before it gets too tight. Something like this might work
```suggestion
            pagination={{ ...props.pagination, hidePageNumbers: props.pagination.pageCount > 4 }}
```

Reply:
thats a good idea, will check with the UX team once before i change this

Reply:
Perfect 👍🏻